Assets/Scripts/SS3D/Systems/Gamemodes/UI/GamemodeObjectivePanelView.cs

```diff
@@ -44,25 +46,10 @@ protected override void OnStart()
             base.OnStart();
 
             _fade.SetFade(false);
-        }
-
-        protected override void HandleUpdate(in float deltaTime)
-        {
-            base.HandleUpdate(in deltaTime);
-
-            ProcessInput();
-        }
 
-        private void ProcessInput()
-        {
-            if (Input.GetKeyDown(KeyCode.Tab))
-            {
-                _fade.SetFade(true);
-            }
-            if (Input.GetKeyUp(KeyCode.Tab))
-            {
-                _fade.SetFade(false);
-            }
+            Controls.OtherActions controls = SystemLocator.Get<InputSystem>().Inputs.Other;
+            controls.Fade.performed += _ => _fade.SetFade(true);
```

Comment:
I would rather avoid lambda expressions in this case, makes things hard to debug if needed later on. Tell me if you disagree. Otherwise replace with good old private methods HandleFadePerformed and HandleFadeCancelled.